### ISSUE TITLE
80% is highest, but not really

### ISSUE THREAD

**Issue by paqpaqpaq** [russtone/homebridge-mi-humidifier#42]
Not sure how to explain this, but if I set the humidifier to 100% in Home, it always will be automatically set back to 80%, which is fine. 

Now, let's assume the humidity is 45% and I set the humidifier in home to 100%, it will automatically be set to 80%. The wheel starts spinning and the blower starts rotating quite fast, just as I expect it to. But when I manually set the humidifier to 3, instead of automatic, it rotates even faster. 

In other words: either the max in the plugin (80%) is not really the highest setting, OR does this have to do with the actual current humidity? E.g., would it spin faster at 80% if the current humidity would be 25%?

**Comment by russtone** [comment]
Hey @paqpaqpaq ,
80% is the max value for target relative humidity parameter of a humidifier (thats why it always set back to 80%).
Automatic mode is implemented in the humidifier firmware not in the plugin, so unfortunately I can't say anything about how it works. The plugin just sets mode parameter of the humidifier to automatic and humidifier itself decides which speed to choose based on some algorithm in firmware.

**Comment by paqpaqpaq** [comment]
Thanks for the swift reply.

So in other words, if I want it to 'boost' humidity, it must be set to 'manual 3', and after that back to 'automatic xx%'?

**Comment by russtone** [comment]
Yes, I think this is the best way to do it
